.github/workflows/frontend.yml

```diff
@@ -7,25 +7,22 @@ jobs:
     steps:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        token: ${{ secrets.CI_TOKEN }}
-    - name: Build Laboratory-Operator image
-      run: cd operators/labInstance-operator/ && docker build -f Dockerfile . 
-      if: github.ref != 'refs/heads/master'
-    - name: Build and Push Laboratory-Operator image
+
+    - name: Website - KubeScore Check
+      run: |
+           docker run -v  ${{ github.workspace }}:/Crownlabs zegl/kube-score score --ignore-test=pod-networkpolicy,container-security-context /Crownlabs/webservice/kubernetes/deployment.yaml /Crownlabs/webservice/kubernetes/svc.yaml /Crownlabs/webservice/kubernetes/ingress-callback.yaml  /Crownlabs/webservice/kubernetes/ingress-content.yaml
```

Comment:
Would it be possible to specify a wildcard such as `/Crownlabs/webservice/kubernetes/*.yaml`? Otherwise I fill we will never update this upon the addition of new manifests.

Reply:
Yeah, I tried with *.yaml in a previous build but it didn't work.

Reply:
I try it again and see if it was another problem.

Reply:
https://github.com/netgroup-polito/CrownLabs/pull/206/checks

Reply:
maybe the ``docker run`` can be used in a bash script that iterates over the ``/Crownlabs/webservice/kubernetes`` directory and groups all the file names in a variable